### ISSUE TITLE
BPA for all semantic models fails if user has access to Admin Monitoring workspace

### ISSUE THREAD

**Issue by DhyanRathore** [microsoft/semantic-link-labs#186]
**Describe the bug**
The semantic model best practice analyzer across all accessible workspaces `run_model_bpa_bulk(workspace=None)` fails if the user can access the Fabric Admin Monitoring workspace. By default, the admin monitoring workspace is assigned to a shared capacity, and moving it to a Fabric capacity didn't work either.

**To Reproduce**
1. Ensure you have access to the [Admin Monitoring](https://learn.microsoft.com/en-us/fabric/admin/monitoring-workspace) workspace
2. In a Fabric notebook, run the semantic model bpa without specifying a workspace (`workspace=None`) or the following snippet
```
%pip install semantic-link-labs
import sempy_labs as labs
import sempy_labs.report as rep
labs.run_model_bpa_bulk(workspace=None)
```

**Expected behavior**
The command should execute successfully, and BPA stats for all the workspaces to which the user has access should be available in the lakehouse table. Alternatively, the Admin Monitoring workspace should be excluded from the scan.

**Screenshots**
![Admin Monitoring on Fabric Capacity](https://github.com/user-attachments/assets/61f50032-e9bc-4748-8ee2-41f04111b4ec)

**Desktop (please complete the following information):**
 - OS: Windows 11 Home build 22631.4249
 - Browser: Firefox 131.0 (x64), Chrome 128.0.6613.178 (x64)

**Additional context**
We do not have the option to turn off the Admin Monitoring workspace. So Fabric Administrators and users with access to the Admin Monitoring workspace can't use the `workspace=None` unless this behavior has been fixed.

**Comment by m-kovalsky** [comment]
Thanks for raising the issue. Made a fix so that the run_model_bpa_bulk will skip the Admin monitoring workspace by skipping workspaces with type 'AdminInsights'

**Comment by m-kovalsky** [comment]
Fixed in [0.8.9](https://github.com/microsoft/semantic-link-labs/releases/tag/0.8.9).
